code/WorkInProgress/Cael_Aislinn/Rust/core_gen.dm

```diff
@@ -78,10 +79,26 @@ max volume of plasma storeable by the field = the total volume of a number of ti
 		assign_uid()
 		id_tag = uid
 
+/obj/machinery/power/rust_core/attackby(obj/item/weapon/W as obj, mob/living/user as mob)
+	. = ..()
+	if(.)
+		return .
```

Comment:
`.` is the default return value if no return value is specified in the return statement, as such you should just write this as `return`